src/components/presentational/CalendarDay/CalendarDay.tsx

```diff
@@ -69,7 +69,7 @@ export default function (props: CalendarDayProps) {
         }
     }
 
-    return <div ref={props.innerRef} className={dayClasses.join(' ')} style={dayStyle} onClick={() => props.onClick(date)}>
+    return <div ref={props.innerRef} className={dayClasses.join(' ')} style={dayStyle} onClick={() => props.onClick ? props.onClick(date) : void(date)}>
```

Comment:
I missed this in the original review of CalendarDay @greinard 

For accessibility, we should not attach onClick to div but instead have a child ```<UnstyledButton>``` element that the onClick is attached to, so that tab completion etc. works appropriately

When there is no "onClick" we would not include the button.

Reply:
👍 https://github.com/CareEvolution/MyDataHelpsUI/pull/176

Reply:
I am going to revert this change in favor of the PR @greinard attached to last comment.